falcon/request.py

```diff
@@ -220,10 +220,9 @@ def __init__(self, env, options=None):
 
         # PERF(kgriffs): if...in is faster than using env.get(...)
         if 'QUERY_STRING' in env:
-            query_str = env['QUERY_STRING']
+            self.query_string = env['QUERY_STRING']
```

Comment:
With this change L230-L231 are no longer necessary. Could you remove those lines?